### PR TITLE
added a priority area

### PR DESCRIPTION
**Pull request by blueskye04**

### CODE REVIEW

**Review by blueskye04 — COMMENTED** [review]
done

File: Front-End/src/Pages.js [hunk 1]
```diff
@@ -135,19 +137,22 @@ function Drivetrain(props) {
...
-                    <br></br>
-
-                    <input type="text" id="LengthInInches" name="drivetrain_LengthInInches" placeholder="Length in Inches" className="text-input" required="required" />
-                    <br></br>
-                    <input type="text" id="PartNumber" name="drivetrain_PartNumber" placeholder="Part Number" className="text-input" required="required" />
-                    <br></br>
-                    <input type="radio" id="Low" name="Low" value="HTML" />
-                    <label for="html">Low</label>
-                    <input type="radio" id="Mid" name="Mid" value="Mid" />
-                    <label for="Mid">Mid</label>
```
Comment (NathanDuPont) [inline Front-End/src/Pages.js:147]: It looks like the "Length in Inches" and "Part Number" inputs got deleted on the Drivetrain page. Was this on purpose?

Drivetrain:
image


Not drivetrain:
image

**Review by NathanDuPont — CHANGES_REQUESTED** [review]
Looks like some inputs were accidentally deleted, + merge conflicts

File: Front-End/src/Pages.js (hunk 1 above)
Comment (blueskye04) [reply Front-End/src/Pages.js]: fixed.

Comment (blueskye04) [reply Front-End/src/Pages.js]: it is fixed

Comment (NathanDuPont) [reply Front-End/src/Pages.js]: perfect. thanks for fixing!

**Review by NathanDuPont — APPROVED** [review]

**Pull request merged**